content/post/articles/Music-and-silence.md

```diff
@@ -0,0 +1,57 @@
+tags: ["Coding","Music","Silence","Software development","CaParleDev","Bugs"]
+authors: ["Shaba-Rex"]
+---
+{{<figure src="/Music-and-silence-in-coding/Listen-to-music.png"  alt="Liste to music while coding">}}
+
+Listen to music while coding or enter the zone (state of concentration created by silence).
+Opinions are divided on the subject, on the one hand we have coders who like to boost their concentration and productivity by listening to music and those who only want "silence"👄, in this article we will explore these two opinions.
+
+# Code with music
+Friedrich Nietzsche said <<*Without music life would be a mistake.*>>
+
+The brain has to make efforts to analyze and process information based on abstract and logical reasoning during coding, several studies such as [University of Birmingham research](http://www.sciencedirect.com/science/article/pii/0003687072901019), [PNAS research](https://www.pnas.org/content/116/8/3310), [Stanford University School of Medicine research](https://www.sciencedirect.com/science/article/pii/S0896627307005004) show that music is effective in improving focus and productivity.
+
+### Concentration and productivity
+The benefits of music on our brain are numerous we can mention:
+###### Boost motivation
+
+I'm sure you know this good old mister "bug" that everyone wants to embrace he only brings us happiness when coding 🐛, more seriously as a developer, we often if not always encounter bugs in our code which can sometimes push us to abandon our code, because we cannot find the error, or because we consider that we have sought too much for the solution, in short it demotivates us but thanks to the studies cited above we can say that to listen a music that we like can help to take a step back, to reconsider the problem from another angle because the music will make us happy, more relaxed and therefore open to new possibilities to kill or permanently solve this dear mister bug 😊.
+
+###### Boost creativity
+
+Creativity can be boosted by music, according to a [study](http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0182210) by researchers at Radboud University (Netherlands) and Sydney University of Technology (Australia) the researchers found that listening to happy music, which they define as music with positive, awakening values (a relative positivity because coders who love devil metal are also happy) facilitates more inspiring creative thinking than silence.
+
+I'm a supporter of this opinion, so here are some nuggets(without wanting to advertise) I use when I meet mister bug:
+- [RISE (ft. The Glitch Mob, Mako, and The Word Alive)](https://www.youtube.com/watch?v=fB8TyLTD7EE)
+- [Phoenix (avec Cailin Russo et Chrissy Costanza)](https://www.youtube.com/watch?v=i1IKnWDecwA)
+- [Take Over (avec Jeremy McKinnon (A Day To Remember), MAX, Henry)](https://www.youtube.com/watch?v=KbNL9ZyB49c)
+
+# Code with silence
+
+Lao Tzu said <<*Silence is a source of Great Strength*>>
```

Comment:
I don't think this quote fits the whole point of this section. Not only [it's not directly from Lao Tzu](https://www.taoistic.com/fake-laotzu-quotes/fake-laotzu-quote-Silence_is_a_source_of_great_strength.htm) but where it's coming from, (The Tao of Leadership: Lao Tzu's Tao Te Ching Adapted for a New Age from 1985, by John Heider), where the author allowed himself considerable deviations from the original. If we quote including the 4 lines before the excerpt you took:

> The leader does not gossip about others or waste breath arguing the merits of competing theories.
Silence is a great source of strength.

Here silence is not meant for silence around you, rather silence from you (or in you as you wish), so I don't think it's really applicable in this specific case.